datumaro/plugins/voc_format/format.py

```diff
@@ -123,42 +125,55 @@ def make_voc_label_map():
     return label_map
 
 
-def parse_label_map(path):
-    if not path:
-        return None
+def parse_label_map(path: str) -> Dict[str, Tuple[RgbColor, List[str], List[str]]]:
+    """
+    Parses a label map file in the format:
+    'name : color (r, g, b) : parts (hand, feet, ...) : actions (siting, standing, ...)'
+
+    Parameters:
+        path: File path
+
+    Returns:
+        A dictionary: label -> (color, parts, actions)
```

Comment:
This is wrong, because the actual return dictionary contains lists, not tuples (the function should probably be fixed, not the description).

Reply:
The code often relies on element modification, moving to tuples would be inefficient for no reason and immutability from tuples is not supposed. It requires refactoring, but it is not in the scope of this PR. Added a comment on this and some type annotations.